content/tutorials/mushr_navigation_system.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "MuSHR_navigation_system_tutorial"
+date: 2021-04-18T22:17:25+05:30
+summary: "This tutorial covers running the MuSHR multi-agent navigation stack in simulation"
+difficulty: "Advanced"
+duration: 60
+featured: false  # whether this is listed at / (must also be top 6 by weight). 
+active: true     # whether this is listed at /tutorials/
+draft: true      # whether Hugo considers this a draft
+weight: 3        # 2 = intro tutorial 3 = anything else
+---
+
+<h2> By: <a href=https://www.sidharthtalia.com/>Sidharth Talia</a></h2>
+
+<!-- Header figure required! -->
+<br>
+{{< figure src="/tutorials/MuSHR_multiagent_navigation/intro.jpg" width="800" >}} <br>                           
+<br>
+
+### Introduction
+A navigation system enables a robot to navigate quickly between different poses while avoiding collisions with the environment or other agents. The navigation system in this project uses the NH-TTC [backend](https://github.com/davisbo/NHTTC), which is a decentralized multi-agent navigation system. The Non-holonomic-Time-To-Collision (NH-TTC) backend considers the Non-holonomic constraints of the car and considers the time to collision with other cars in the optimization process. The reader may peruse the [paper](http://motion.cs.umn.edu/r/NH-TTC/arxiv-NHTTC.pdf) to better understand how it works.
+
+### Goal
+The goal of this tutorial is to get the reader familiar with the multi-agent navigation framework used for the MuSHR car.
+
+### Requirements
+Completed the [quick-start](https://mushr.io/tutorials/quickstart/) tutorial (familiarity with ROS and python is assumed).
+
+## Environment setup
+Clone the nhttc_ros repository into your catkin workspace: 
+{{< highlight bash >}}
+$ cd catkin_ws/src
+$ git clone --branch devel https://github.com/naughtyStark/nhttc_ros.git
+$ cd nhttc_ros
+$ git submodule init
+$ git submodule update --force --recursive --init --remote
+{{< / highlight >}}
+
+Compile using catkin_make:
+{{< highlight bash >}}
+$ cd ~/catkin_mushr
+$ catkin_make
+{{< / highlight >}}
+
+If everything compiles, you should be ready to try out the simulation example. Launch the nhttc_demo.launch:
+{{< highlight bash >}}
+$ roslaunch nhttc_ros nhttc_demo.launch
+{{< / highlight >}}
+
+Wait till you see the highlighted text on the terminal:
+<br>
+{{< figure src="/tutorials/MuSHR_multiagent_navigation/start-up-message.jpg" width="800" >}} <br>                           
+<br>
+
+In a new tab, open rviz:
+{{< highlight bash >}}
+$ rosrun rviz rviz
+{{< / highlight >}}
+
+Select the rviz configuration corresponding to the nhttc_ros. Run the route publisher node:
+{{< highlight bash >}}
+$ rosrun nhttc_ros route_publisher.py
+{{< / highlight >}}
+
+You should see something like this on rviz:
+<br>
+{{< figure src="/tutorials/MuSHR_multiagent_navigation/demo_default.gif" width="800" >}} <br>                           
```

Comment:
nhttc.rviz does not have the same background config.

Reply:
It appears the same in my system unless you're talking about the zoom in? I zoomed in so that the cars would be more clearly visible. I can change that in the rviz config.

Reply:
fixed (I think?)